### ISSUE TITLE
Error: dylib does not exist

### ISSUE THREAD

**Issue by haydendavenport** [AndrewBelt/NotStraightLines#1]
I'm new to VCV Rack. Does that mean Mac is not supported, or did I download something incorrectly?

Thanks!

**Comment by AndrewBelt** [comment]
Did you compile it?

**Comment by haydendavenport** [comment]
Nope! I've only downloaded prebuilt plugins, so that explains that--I'll look up how to build it tomorrow. Thanks for the help! ^-^
